packages/paste-core/components/page-header/stories/index.stories.tsx

```diff
@@ -84,6 +86,81 @@ export const Default = (): React.ReactElement => {
   );
 };
 
+export const SeparatorVersusNav = (): React.ReactElement => {
+  return (
+    <Box display="flex" columnGap="space200">
+      <Box width="size70" backgroundColor="colorBackground" height="min-content">
+        <PageHeader size="default">
+          <PageHeaderSetting>
+            <Breadcrumb>
+              <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+              <BreadcrumbItem href="#">Breadcrumb</BreadcrumbItem>
+            </Breadcrumb>
+          </PageHeaderSetting>
+          <PageHeaderDetails>
+            <PageHeaderKeyword>Wizard title</PageHeaderKeyword>
+            <PageHeaderPrefix>
+              <Avatar size="sizeIcon90" name="Avatar Name" />
+            </PageHeaderPrefix>
+            <PageHeaderHeading>Page title</PageHeaderHeading>
+            <PageHeaderActions>
+              <Button variant="secondary">Action</Button>
+            </PageHeaderActions>
+            <PageHeaderMeta>
+              <Badge variant="success" as="span">
+                Badge
+              </Badge>
+              Meta
+              <Anchor href="#">Meta</Anchor>
+            </PageHeaderMeta>
+            <PageHeaderParagraph>Paragraph text</PageHeaderParagraph>
+          </PageHeaderDetails>
+          <PageHeaderSeparator>
+            <Separator orientation="horizontal" />
+          </PageHeaderSeparator>
```

Comment:
question: does this fall victim to the redundancy problem again? A separator in a seperator?

Reply:
It follows the pattern of the InPageNav API

<PageHeaderInPageNavigation>
  <InPageNavigation />
</PageHeaderInPageNavigation>

Reply:
```
<PageHeaderInPageNavigation>
  <InPageNavigation />
</PageHeaderInPageNavigation>
```